src/main/java/de/tum/in/www1/artemis/domain/metis/Post.java

```diff
@@ -61,6 +61,9 @@
     @Column(name = "course_wide_context")
     private CourseWideContext courseWideContext;
 
+    @ManyToOne
+    private Conversation conversation;
```

Comment:
Because we need to list all conversations of a user while at the messages page, but this does not mean that all posts of each conversation must be fetched. We simply don't need this.

Reply:
> We simply don't need this.

You actually have a use case for this, so it makes sense to have it.

Reply:
I decided not to have a counterpart because I want to stick to the way posts are fetched at the discussion page, since the developed page uses the previous implementations of METIS in order to avoid code clones.

Reply:
I see what you mean. Currently, however I do not have a use case for accessing posts over a conversation, even on the server-side. However, if this is the Artemis convention, I can add the counterpart, mark it with JsonIgnore, and lazy fetching if needed. Please let me know if this is the case.